.travis.yml

```diff
@@ -61,11 +61,13 @@ matrix:
           env: PYTHON_VERSION=2.7 SETUP_CMD='build_docs -w'
                PIP_DEPENDENCIES='gwcs'
                CONDA_DEPENDENCIES='Cython scipy scikit-image scikit-learn matplotlib jupyter'
```

Comment:
I think moving gwcs to the CONDA_DEPENDENCIES out of PIP_DEPENDENCIES should fix the failures, at least it worked when I was testing this here before the helpers release. (The failures are due that the released version is not yet using helpers 2.0, thus is running into the same incompatibilities with typing as any other package when you try to build it.)